lib/fluent/output.rb

```diff
@@ -98,7 +98,7 @@ class OutputThread
     def initialize(output)
       @output = output
       @finish = false
-      @next_time = Engine.now + 1.0
+      @next_time = Time.now.to_f + 1.0
```

Comment:
What about just directly use `Time.now + 1.0`.  Time objects can be compared or subtracted as-is.

```ruby
irb(main):001:0> next_time = Time.now + 1.0
=> 2015-03-28 11:24:03 +0900
irb(main):002:0> next_wait = next_time - Time.now
=> 0.719691
```